apps/explorer/src/services/helpers/tryGetOrderOnAllNetworks.ts

```diff
@@ -23,17 +23,15 @@ export type GetOrderApi<T, R> = {
   defaultParams: GetOrderParamsApi<T>
 }
 
-type TypeOrderApiParams = GetOrderParams | GetTxOrdersParams
-
 export async function tryGetOrderOnAllNetworksAndEnvironments<TypeOrderResult>(
   networkId: Network,
-  getOrderApi: GetOrderApi<TypeOrderApiParams, TypeOrderResult>,
-  networkIdSearchListRemaining: Network[] = NETWORK_ID_SEARCH_LIST,
+  getOrderApi: GetOrderApi<GetOrderParams, TypeOrderResult> | GetOrderApi<GetTxOrdersParams, TypeOrderResult>,
+  networkIdSearchListRemaining: Network[] = NETWORK_ID_SEARCH_LIST
 ): Promise<GetOrderResult<TypeOrderResult>> {
   // Get order
-  let order = null
+  let order: TypeOrderResult | null = null
   try {
-    order = await getOrderApi.api({ ...getOrderApi.defaultParams, networkId })
+    order = await getOrderApi.api({ ...getOrderApi.defaultParams, networkId } as never)
```

Comment:
Why as `never`?

Reply:
<img width="1281" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/3c0957c9-f8c3-4ea8-997d-9659d7adfe69">

Due to typescript error. I'm afraid of changing the logic here.
I would prefer keep it as it is + TODO to fix types in the future